TaskManagement/timeline/templatetags/task_count_tags.py

```diff
@@ -0,0 +1,43 @@
+from django import template
+from task.models import Task
+from project.models import Project, ProjectTeam, Role
+from task.constants import STATUS
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_task_count(proj_id):
+    return Task.objects.filter(project__id=proj_id.id).count()
+
+
+@register.simple_tag
+def get_task_status_count(proj_id):
+    result = {}
+    for status in STATUS:
+        result[status[0]] = Task.objects.filter(project__id=proj_id.id, status=status[0]).count()
+    return result
+
+
+@register.simple_tag
+def get_team_size(proj_id):
```

Comment:
write tags related to tasks here in task app and tags related to project should be in project app.